### DISCUSSION TITLE
expanding copilot to know about nim-lang

### DISCUSSION THREAD

**Discussion by alsoeric** [community/community#40171] [Copilot Conversations]
### Select Topic Area

Question

### Body

 what would it take to build nim into the co-pilot training model?

tl;dr
Have to confess to being a bit of a co-pilot fanboy. My career as a software developer ended after 18 years in 1994 because of repetitive strain injury. Now with co-pilot and speech recognition, I can create code almost identical to what I wrote myself when my hands worked.

The downside is that it's hard to explorer new language environments. For example I'm happy with the python model. It works well handles type hints and generally does all the right things. I've seen enough ratings on nim it looks like a language with exploring but without the accessibility provided by Copilot, I'm not doing much with it.  Hence my question: what would it take to build nim into the co-pilot training model?

**Comment by ossanna16** [comment]
Hi there @alsoeric and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
